app/models/converters/school_rating_converter.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SchoolRatingConverter < BaseConverter
+  def self.convert(value)
+    # allow non-numeric values and let validation handle it
+    return value unless value.is_a? Numeric
+
+    value = value.to_i
+    value = 5 if value > 5
```

Comment:
take it or leave it, but written like this you can sometimes avoid the second check:
```
if value > 5
  value = 5
elsif value <= 0 # or < 1
 value = nil
end